governance.md

```diff
@@ -0,0 +1,175 @@
+# The United States Research Software Engineer (US-RSE) Association
+
+## Membership and Governance
+
+Date: June XX, 2020
+Authors: The founding members of the US-RSE Association Steering Committee
```

Comment:
should we list our names here?

Reply:
That seems fine by me.

Reply:
Having a list of authors could become cumbersome over time, especially since we've allowed revisions to this document. Is there a need to have an author list?

Reply:
If you want to just remove the line "Authors: The founding members of the US-RSE Association Steering Committee", that would be fine with me

Reply:
Now that you bring it up, I agree that it could get cumbersome. I'm fine with removing the "Authors: .." line too.